Api/Operation/OperationFactory.php

```diff
@@ -46,11 +47,12 @@ public function createCollectionOperation(
         ResourceInterface $resource,
         $methods,
         $path = null,
+        $requirements = [],
```

Comment:
Can you move it at the end of the method. It's currently a BC break and according to semver we don't allow BC breaks in the 1.x branch.